v3.0/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -238,15 +235,62 @@ Check Table Before Drop: false
 1 row in set (0.001 sec)
 ```
 
+## Manage multiple TiDB clusters
+
+An instance of a `tidb-cluster` module corresponds to a TiDB cluster in the GKE cluster. If you want to add a new TiDB cluster,
+you can edit the `tidbclusters.tf` file and add a new instance of the `tidb-cluster` module. For example:
+
+```hcl
+module "example-tidb-cluster" {
+  providers = {
+    helm = "helm.gke"
+  }
+  source                     = "../modules/gcp/tidb-cluster"
+  gcp_project                = "gcp-project-name"
+  gke_cluster_location       = "us-west1"
+  gke_cluster_name           = "gke-cluster-name"
+  cluster_name               = "example-tidb-cluster"
+  cluster_version            = "v3.0.1"
+  kubeconfig_path            = module.tidb-operator.kubeconfig_path
+  tidb_cluster_chart_version = "v1.0.0"
+  pd_instance_type           = "n1-standard-1"
+  tikv_instance_type         = "n1-standard-4"
+  tidb_instance_type         = "n1-standard-2"
+  monitor_instance_type      = "n1-standard-1"
+  pd_node_count              = 1
+  tikv_node_count            = 2
+  tidb_node_count            = 1
+  monitor_node_count         = 1
+}
+```
+
+> **Note:**
+>
+> The `cluster_name` of each cluster must be unique.
+
+> **Note:**
```

Comment:
@yikeke The two notes here need to be combined together, right?

Reply:
Yes, avoiding two consecutive notes improves readability. In this particular case, you can use:

```
> **Note:**
>
> - The `cluster_name` of each cluster must be unique.
>
> - The actual number of nodes created are multiplied by the number of availability zones in the region.
```